src/pages/_app.tsx

```diff
@@ -2,8 +2,16 @@ import 'css/global.css'
 import { AppProps } from 'next/app'
 import { useAppGA } from 'lib/ga'
 import { AppContextProvider } from 'context/app'
+import { useMultiKeyPressTrap } from 'hooks/use-multi-key-press'
+import { isDev } from 'lib/constants'
 
 const App = ({ Component, pageProps }: AppProps) => {
+  if (isDev) {
+    useMultiKeyPressTrap(['control+i', 'alt+i'], () =>
+      document.body.classList.toggle('inspect')
```

Comment:
y si en vez de esto metemos [inspx](https://github.com/raunofreiberg/inspx)? 😮

Reply:
also, haría q se pueda usar también en preview. con `NEXT_PUBLIC_VERCEL_ENV `

Reply:
Inspx podría ser complementario. Hay veces que no sabes bien cual es el elemento que esta causando problemas y puede ser util ver todos los box de una. Also, lo encontré un poco buggy con los key bindings en windows 😬.

Reply:
k — dejemos este por ahora. ni hagamos preview tampoco. [see this](https://github.com/basementstudio/next-typescript/pull/170#discussion_r716117303). mejor no agregar ruido en preview.

Reply:
(sorry for the back n forth)

Reply:
Si vemos que no es muy útil en el uso lo volamos. WDYT?

Reply:
ship it.

Reply:
Joya, actualizo los key traps por la lib que me pasaste que seguro performa mejor.